src/main/scala/nl/biopet/utils/Histogram.scala

```diff
@@ -65,7 +63,7 @@ class Histogram[T](_counts: Map[T, Long] = Map[T, Long]())(
   /** Write histogram to a tsv/count file */
   def writeAggregateToTsv(file: File): Unit = {
     val writer = new PrintWriter(file)
-    aggregateStats.foreach(x => writer.println(x._1 + "\t" + x._2))
+    aggregateStats.foreach { case (k, v) => writer.println(s"$k\t$v") }
```

Comment:
Can you give a more descriptive name for k and v?
The whole criticism codacy gives in these cases is that `x._1` and `x._2` are non-descriptive and therefore it is hard to see what the code is doing.
This is not solved by replacing them with equally non-descript variables `key` and `value`.